### ISSUE TITLE
Aliasing with `@` is not recognized as 'internal'

### ISSUE THREAD

**Issue by ombene** [import-js/eslint-plugin-import#2333]
This seems to be a repeat of an issue that has been resolved and regressed twice now, see https://github.com/import-js/eslint-plugin-import/issues/2145 and https://github.com/import-js/eslint-plugin-import/issues/2249.

In my case the source code is private, but we're using a config that looks like:

```jsonc
// ...
"import/resolver": {
  "alias": [
    ["@", "./src"]
  ]
}
// ...
```

(See also https://github.com/johvin/eslint-import-resolver-alias.) And with source code like:

```jsx
// ...
import exampleInternalViaAlias from '@/exampleInternalViaAlias';

import exampleSibling from './exampleSibling';
// ...
```

When upgrading to v2.25.3 from v2.22.1, without any other changes to source code or other dependencies, we have now started getting false negatives like:

```
  3:1  error  `@/exampleInternalViaAlias` import should occur after import of `./exampleSibling`  import/order
```

Injecting some logs into the installed version of `eslint-plugin-import` in our node_modules, I can see that the import type is getting calculated as `'unknown'` (instead of `'internal'`) for `'@/exampleInternalViaAlias'`.
